packages/fastfetch/build.sh

```diff
@@ -1,12 +1,12 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/fastfetch-cli/fastfetch
-TERMUX_PKG_DESCRIPTION="A neofetch-like tool for fetching system information and displaying them in a pretty way"
+TERMUX_PKG_DESCRIPTION="A maintained, feature-rich and performance oriented, neofetch like system information tool"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="2.34.0"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/fastfetch-cli/fastfetch/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=2dae260d604557c6bf5a9781b3c960aa4265a067e54a984f8c68889a578193a9
-TERMUX_PKG_DEPENDS="vulkan-loader"
-TERMUX_PKG_BUILD_DEPENDS="freetype, libandroid-wordexp-static, vulkan-headers, vulkan-loader-android"
+TERMUX_PKG_BUILD_DEPENDS="freetype, libandroid-wordexp-static, vulkan-headers, vulkan-loader-android, ocl-icd, opencl-headers, mesa-dev"
```

Comment:
```suggestion
TERMUX_PKG_BUILD_DEPENDS="freetype, libandroid-wordexp-static, mesa-dev, ocl-icd, opencl-headers, vulkan-headers, vulkan-loader-generic"
```
`vulkan-loader-android` will be removed in the future so use `vulkan-loader-generic` for building

Reply:
> `vulkan-loader-android` will be removed in the future

Probably we already can replace it with xMeM's mesa-vulkan-icd-layer. I suppose it works fine for non-x11 stuff, x11 support is fine on one of my devices.